Dockerfile

```diff
@@ -15,12 +19,20 @@ RUN apt-get update && apt-get install -y \
     build-essential \
     libasound2-dev
 
-RUN wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
-    chmod +x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
-    ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh --user && \
-    echo "PATH=/root/bin:$PATH\n" >> /root/.bashrc && \
-    echo "source /root/.bazel/bin/bazel-complete.bash" >> /root/.bashrc && \
-    rm ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+RUN if [ "$TARGET_ARCH" = "amd64" ]; then \
+        wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
+        chmod +x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
+        ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh --user && \
+        echo "PATH=/root/bin:$PATH\n" >> /root/.bashrc && \
+        echo "source /root/.bazel/bin/bazel-complete.bash" >> /root/.bashrc && \
+        rm ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh; \
+    elif [ "$TARGET_ARCH" = "arm64" ]; then \
+        apt-get update && apt-get install -y --no-install-recommends openjdk-11-jdk-headless && \
+        wget https://urm.nvidia.com/artifactory/sw-ai-app-bazel/sw-ai-app-jarvis/tools_aarch64/bazel_aarch64_${BAZEL_VERSION} && \
```

Comment:
Does this need to be external url

Reply:
Yes good catch. Will need to check where we could upload this.